cs/v1/CS_v1.0.0.yaml

```diff
@@ -165,6 +165,20 @@ paths:
           name: deliveryTypeAtDestination
           description: Indicates the type of service offered at Destination. Options are defined in the `Receipt/Delivery` entity.  - CY(Container yard (incl. rail ramp))  - SD (Store Door)  - CFS (Container Freight Station)
         - $ref: ../../models/CommonComponents.yaml#/components/parameters/APIVersionMajor
+        - schema:
+            type: integer
+            format: int32
+            default: 100
+            example: 100
+            minimum: 1
+          in: query
+          name: limit
+          description: Maximum number of items to return.
+        - schema:
+            type: string
```

Comment:
Do we want a `maxLength` here? (I am not sure what it should be)

Reply:
For the cursor?

Reply:
yes

Reply:
Do we have maxlength for cursor in other APIs?